### PR TITLE
Rename real_topK_per_nq_ to topks_per_nq_prefix_sum_ in SearchResult

### PR DESCRIPTION
**Pull request by cydrain**
Signed-off-by: yudong.cai <yudong.cai@zilliz.com>
Issue: #17599

### CODE REVIEW

**Comment by cydrain** [comment]
rerun ut

File: internal/core/src/segcore/Reduce.cpp [hunk 1]
```diff
@@ -110,6 +112,7 @@ ReduceHelper::Reduce() {
 
 void
 ReduceHelper::Marshal() {
+#if 0
```
Comment (bigsheeper) [inline internal/core/src/segcore/Reduce.cpp:115]: Just remove old codes?

Comment (cydrain) [reply internal/core/src/segcore/Reduce.cpp:115]: yes

**Review by congqixia — APPROVED** [review]
/approve

**Pull request merged**